test/test_automl/test_automl.py

```diff
@@ -43,8 +51,24 @@ def setUp(self):
 
         self.automl._seed = 42
         self.automl._backend = unittest.mock.Mock(spec=Backend)
+        self.automl._backend.context = unittest.mock.Mock()
         self.automl._delete_output_directories = lambda: 0
 
+    @classmethod
+    def setUpClass(self):
+        dask.config.set({'distributed.worker.daemon': False})
+        self.client = dask.distributed.Client(
+            dask.distributed.LocalCluster(
+                n_workers=1,
+                processes=False,
+                threads_per_worker=2,
```

Comment:
Does this mean all tests are now executed with two workers in parallel?

Reply:
I have created a fixture so that a global client is created once, and we supply task to this unique client. 

With multiple clients, pytest seem to struggle with:
-> Logging: I think some stdout/stderror capturing is redirected from pytest, and with multiple clients there are multiple threadlocking jobs that at the end of the pytest run are simply not closed. No matter if I call client.shutdown, client.close.


It is cleaner to have a single client which we can track for debug if needed.